packages/api/tests/lib/writeRecords/test-write-granules.js

```diff
@@ -575,12 +575,13 @@ test.serial('writeGranulesFromMessage() saves granule records to Dynamo/PostgreS
   );
 
   const esRecord = await t.context.esGranulesClient.get(granuleId);
-
+  t.is(granulePgRecord.timestamp.getTime(), dynamoRecord.timestamp);
```

Comment:
These tests should probably also do a `t.truthy(dynamoRecord.timestamp)` because even if the value is the same in all the databases it still could be an empty value

Reply:
eb5bbc304d